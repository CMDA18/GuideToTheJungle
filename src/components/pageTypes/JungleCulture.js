// @flow
import React, { Component } from 'react'
import Helmet from 'react-helmet'
// import styled from 'styled-components'
import Grid from '../Grid'
import PageTitle from '../PageTitle'
import Intro from '../Intro'
import TextAndImage from '../TextAndImage'
import { jungleCulture } from '../../constants/content/index'

type Props = {
  test: string
}

class JungleCulture extends Component<Props> {
  render () {
    return (
      <div className='JungleCulture' data-test={this.props.test}>
        <Helmet
          title='JungleCulture'
        />
        <Grid page='jungleCulture'>
          <header>
            <PageTitle {...jungleCulture}/>
            <Intro {...jungleCulture}/>
          </header>
          <TextAndImage {...jungleCulture} />
        </Grid>

      </div>
    )
  };
}

export default JungleCulture
