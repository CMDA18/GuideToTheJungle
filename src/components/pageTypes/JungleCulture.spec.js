import React from 'react'
import { shallow } from 'enzyme'
import JungleCulture from './JungleCulture'

it('renders without crashing', () => {
  expect(shallow(<JungleCulture />)).toMatchSnapshot()
})
