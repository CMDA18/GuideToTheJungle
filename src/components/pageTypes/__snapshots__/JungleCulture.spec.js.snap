// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <JungleCulture />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <HelmetWrapper
          defer={true}
          encodeSpecialCharacters={true}
          title="JungleCulture"
        />,
        <Grid
          page="jungleCulture"
        >
          <header>
            <PageTitle
              image="./images/team_design.jpg"
              imageAlignment="left"
              introText="Welkom tijdens jouw tweede week binnen Jungle Minds. Deze week staat in de teken van ons cultuur. Tijdens deze module leer je hoe wij ons cultuur gevormd hebben en hoe wij deze doorvertallen aan onze medewerkers."
              introTitle="Module 2"
              mediaCaption="Jungle Cultuur"
              pageTitle="2 De Jungle Cultuur"
              textMedia={
                Array [
                  Object {
                    "text": "Tijdens deze week leer je meer over:
      <ul>
      <li>
        2.1 Jungle Formula
      </li>
      <li>
        2.2 Jungle Days
      </li>
      <li>
        2.3 ...
      </li>
      </ul>
      ",
                    "title": "Week 2",
                  },
                  Object {
                    "text": "Gedurende deze modules krijg je enkele opdrachten toegewegen. Het is de bedoeling dat je deze opdrachten uitvoert voor het einde van de week. Denk aan: de mandagen zijn de beste dagen om een gesprek aan te gaan met een collega. 
      <ul>
      <li>
        2.4 Opdrachten
      </li>
      </ul>
      ",
                    "title": "Opdrachten",
                  },
                ]
              }
            />
            <Intro
              image="./images/team_design.jpg"
              imageAlignment="left"
              introText="Welkom tijdens jouw tweede week binnen Jungle Minds. Deze week staat in de teken van ons cultuur. Tijdens deze module leer je hoe wij ons cultuur gevormd hebben en hoe wij deze doorvertallen aan onze medewerkers."
              introTitle="Module 2"
              mediaCaption="Jungle Cultuur"
              pageTitle="2 De Jungle Cultuur"
              textMedia={
                Array [
                  Object {
                    "text": "Tijdens deze week leer je meer over:
      <ul>
      <li>
        2.1 Jungle Formula
      </li>
      <li>
        2.2 Jungle Days
      </li>
      <li>
        2.3 ...
      </li>
      </ul>
      ",
                    "title": "Week 2",
                  },
                  Object {
                    "text": "Gedurende deze modules krijg je enkele opdrachten toegewegen. Het is de bedoeling dat je deze opdrachten uitvoert voor het einde van de week. Denk aan: de mandagen zijn de beste dagen om een gesprek aan te gaan met een collega. 
      <ul>
      <li>
        2.4 Opdrachten
      </li>
      </ul>
      ",
                    "title": "Opdrachten",
                  },
                ]
              }
            />
          </header>
          <TextAndImage
            image="./images/team_design.jpg"
            imageAlignment="left"
            introText="Welkom tijdens jouw tweede week binnen Jungle Minds. Deze week staat in de teken van ons cultuur. Tijdens deze module leer je hoe wij ons cultuur gevormd hebben en hoe wij deze doorvertallen aan onze medewerkers."
            introTitle="Module 2"
            mediaCaption="Jungle Cultuur"
            pageTitle="2 De Jungle Cultuur"
            textMedia={
              Array [
                Object {
                  "text": "Tijdens deze week leer je meer over:
      <ul>
      <li>
        2.1 Jungle Formula
      </li>
      <li>
        2.2 Jungle Days
      </li>
      <li>
        2.3 ...
      </li>
      </ul>
      ",
                  "title": "Week 2",
                },
                Object {
                  "text": "Gedurende deze modules krijg je enkele opdrachten toegewegen. Het is de bedoeling dat je deze opdrachten uitvoert voor het einde van de week. Denk aan: de mandagen zijn de beste dagen om een gesprek aan te gaan met een collega. 
      <ul>
      <li>
        2.4 Opdrachten
      </li>
      </ul>
      ",
                  "title": "Opdrachten",
                },
              ]
            }
          />
        </Grid>,
      ],
      "className": "JungleCulture",
      "data-test": undefined,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "defer": true,
          "encodeSpecialCharacters": true,
          "title": "JungleCulture",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": Array [
            <header>
              <PageTitle
                image="./images/team_design.jpg"
                imageAlignment="left"
                introText="Welkom tijdens jouw tweede week binnen Jungle Minds. Deze week staat in de teken van ons cultuur. Tijdens deze module leer je hoe wij ons cultuur gevormd hebben en hoe wij deze doorvertallen aan onze medewerkers."
                introTitle="Module 2"
                mediaCaption="Jungle Cultuur"
                pageTitle="2 De Jungle Cultuur"
                textMedia={
                  Array [
                    Object {
                      "text": "Tijdens deze week leer je meer over:
      <ul>
      <li>
        2.1 Jungle Formula
      </li>
      <li>
        2.2 Jungle Days
      </li>
      <li>
        2.3 ...
      </li>
      </ul>
      ",
                      "title": "Week 2",
                    },
                    Object {
                      "text": "Gedurende deze modules krijg je enkele opdrachten toegewegen. Het is de bedoeling dat je deze opdrachten uitvoert voor het einde van de week. Denk aan: de mandagen zijn de beste dagen om een gesprek aan te gaan met een collega. 
      <ul>
      <li>
        2.4 Opdrachten
      </li>
      </ul>
      ",
                      "title": "Opdrachten",
                    },
                  ]
                }
              />
              <Intro
                image="./images/team_design.jpg"
                imageAlignment="left"
                introText="Welkom tijdens jouw tweede week binnen Jungle Minds. Deze week staat in de teken van ons cultuur. Tijdens deze module leer je hoe wij ons cultuur gevormd hebben en hoe wij deze doorvertallen aan onze medewerkers."
                introTitle="Module 2"
                mediaCaption="Jungle Cultuur"
                pageTitle="2 De Jungle Cultuur"
                textMedia={
                  Array [
                    Object {
                      "text": "Tijdens deze week leer je meer over:
      <ul>
      <li>
        2.1 Jungle Formula
      </li>
      <li>
        2.2 Jungle Days
      </li>
      <li>
        2.3 ...
      </li>
      </ul>
      ",
                      "title": "Week 2",
                    },
                    Object {
                      "text": "Gedurende deze modules krijg je enkele opdrachten toegewegen. Het is de bedoeling dat je deze opdrachten uitvoert voor het einde van de week. Denk aan: de mandagen zijn de beste dagen om een gesprek aan te gaan met een collega. 
      <ul>
      <li>
        2.4 Opdrachten
      </li>
      </ul>
      ",
                      "title": "Opdrachten",
                    },
                  ]
                }
              />
            </header>,
            <TextAndImage
              image="./images/team_design.jpg"
              imageAlignment="left"
              introText="Welkom tijdens jouw tweede week binnen Jungle Minds. Deze week staat in de teken van ons cultuur. Tijdens deze module leer je hoe wij ons cultuur gevormd hebben en hoe wij deze doorvertallen aan onze medewerkers."
              introTitle="Module 2"
              mediaCaption="Jungle Cultuur"
              pageTitle="2 De Jungle Cultuur"
              textMedia={
                Array [
                  Object {
                    "text": "Tijdens deze week leer je meer over:
      <ul>
      <li>
        2.1 Jungle Formula
      </li>
      <li>
        2.2 Jungle Days
      </li>
      <li>
        2.3 ...
      </li>
      </ul>
      ",
                    "title": "Week 2",
                  },
                  Object {
                    "text": "Gedurende deze modules krijg je enkele opdrachten toegewegen. Het is de bedoeling dat je deze opdrachten uitvoert voor het einde van de week. Denk aan: de mandagen zijn de beste dagen om een gesprek aan te gaan met een collega. 
      <ul>
      <li>
        2.4 Opdrachten
      </li>
      </ul>
      ",
                    "title": "Opdrachten",
                  },
                ]
              }
            />,
          ],
          "page": "jungleCulture",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <PageTitle
                  image="./images/team_design.jpg"
                  imageAlignment="left"
                  introText="Welkom tijdens jouw tweede week binnen Jungle Minds. Deze week staat in de teken van ons cultuur. Tijdens deze module leer je hoe wij ons cultuur gevormd hebben en hoe wij deze doorvertallen aan onze medewerkers."
                  introTitle="Module 2"
                  mediaCaption="Jungle Cultuur"
                  pageTitle="2 De Jungle Cultuur"
                  textMedia={
                    Array [
                      Object {
                        "text": "Tijdens deze week leer je meer over:
      <ul>
      <li>
        2.1 Jungle Formula
      </li>
      <li>
        2.2 Jungle Days
      </li>
      <li>
        2.3 ...
      </li>
      </ul>
      ",
                        "title": "Week 2",
                      },
                      Object {
                        "text": "Gedurende deze modules krijg je enkele opdrachten toegewegen. Het is de bedoeling dat je deze opdrachten uitvoert voor het einde van de week. Denk aan: de mandagen zijn de beste dagen om een gesprek aan te gaan met een collega. 
      <ul>
      <li>
        2.4 Opdrachten
      </li>
      </ul>
      ",
                        "title": "Opdrachten",
                      },
                    ]
                  }
                />,
                <Intro
                  image="./images/team_design.jpg"
                  imageAlignment="left"
                  introText="Welkom tijdens jouw tweede week binnen Jungle Minds. Deze week staat in de teken van ons cultuur. Tijdens deze module leer je hoe wij ons cultuur gevormd hebben en hoe wij deze doorvertallen aan onze medewerkers."
                  introTitle="Module 2"
                  mediaCaption="Jungle Cultuur"
                  pageTitle="2 De Jungle Cultuur"
                  textMedia={
                    Array [
                      Object {
                        "text": "Tijdens deze week leer je meer over:
      <ul>
      <li>
        2.1 Jungle Formula
      </li>
      <li>
        2.2 Jungle Days
      </li>
      <li>
        2.3 ...
      </li>
      </ul>
      ",
                        "title": "Week 2",
                      },
                      Object {
                        "text": "Gedurende deze modules krijg je enkele opdrachten toegewegen. Het is de bedoeling dat je deze opdrachten uitvoert voor het einde van de week. Denk aan: de mandagen zijn de beste dagen om een gesprek aan te gaan met een collega. 
      <ul>
      <li>
        2.4 Opdrachten
      </li>
      </ul>
      ",
                        "title": "Opdrachten",
                      },
                    ]
                  }
                />,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "image": "./images/team_design.jpg",
                  "imageAlignment": "left",
                  "introText": "Welkom tijdens jouw tweede week binnen Jungle Minds. Deze week staat in de teken van ons cultuur. Tijdens deze module leer je hoe wij ons cultuur gevormd hebben en hoe wij deze doorvertallen aan onze medewerkers.",
                  "introTitle": "Module 2",
                  "mediaCaption": "Jungle Cultuur",
                  "pageTitle": "2 De Jungle Cultuur",
                  "textMedia": Array [
                    Object {
                      "text": "Tijdens deze week leer je meer over:
      <ul>
      <li>
        2.1 Jungle Formula
      </li>
      <li>
        2.2 Jungle Days
      </li>
      <li>
        2.3 ...
      </li>
      </ul>
      ",
                      "title": "Week 2",
                    },
                    Object {
                      "text": "Gedurende deze modules krijg je enkele opdrachten toegewegen. Het is de bedoeling dat je deze opdrachten uitvoert voor het einde van de week. Denk aan: de mandagen zijn de beste dagen om een gesprek aan te gaan met een collega. 
      <ul>
      <li>
        2.4 Opdrachten
      </li>
      </ul>
      ",
                      "title": "Opdrachten",
                    },
                  ],
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "image": "./images/team_design.jpg",
                  "imageAlignment": "left",
                  "introText": "Welkom tijdens jouw tweede week binnen Jungle Minds. Deze week staat in de teken van ons cultuur. Tijdens deze module leer je hoe wij ons cultuur gevormd hebben en hoe wij deze doorvertallen aan onze medewerkers.",
                  "introTitle": "Module 2",
                  "mediaCaption": "Jungle Cultuur",
                  "pageTitle": "2 De Jungle Cultuur",
                  "textMedia": Array [
                    Object {
                      "text": "Tijdens deze week leer je meer over:
      <ul>
      <li>
        2.1 Jungle Formula
      </li>
      <li>
        2.2 Jungle Days
      </li>
      <li>
        2.3 ...
      </li>
      </ul>
      ",
                      "title": "Week 2",
                    },
                    Object {
                      "text": "Gedurende deze modules krijg je enkele opdrachten toegewegen. Het is de bedoeling dat je deze opdrachten uitvoert voor het einde van de week. Denk aan: de mandagen zijn de beste dagen om een gesprek aan te gaan met een collega. 
      <ul>
      <li>
        2.4 Opdrachten
      </li>
      </ul>
      ",
                      "title": "Opdrachten",
                    },
                  ],
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": "header",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "image": "./images/team_design.jpg",
              "imageAlignment": "left",
              "introText": "Welkom tijdens jouw tweede week binnen Jungle Minds. Deze week staat in de teken van ons cultuur. Tijdens deze module leer je hoe wij ons cultuur gevormd hebben en hoe wij deze doorvertallen aan onze medewerkers.",
              "introTitle": "Module 2",
              "mediaCaption": "Jungle Cultuur",
              "pageTitle": "2 De Jungle Cultuur",
              "textMedia": Array [
                Object {
                  "text": "Tijdens deze week leer je meer over:
      <ul>
      <li>
        2.1 Jungle Formula
      </li>
      <li>
        2.2 Jungle Days
      </li>
      <li>
        2.3 ...
      </li>
      </ul>
      ",
                  "title": "Week 2",
                },
                Object {
                  "text": "Gedurende deze modules krijg je enkele opdrachten toegewegen. Het is de bedoeling dat je deze opdrachten uitvoert voor het einde van de week. Denk aan: de mandagen zijn de beste dagen om een gesprek aan te gaan met een collega. 
      <ul>
      <li>
        2.4 Opdrachten
      </li>
      </ul>
      ",
                  "title": "Opdrachten",
                },
              ],
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <HelmetWrapper
            defer={true}
            encodeSpecialCharacters={true}
            title="JungleCulture"
          />,
          <Grid
            page="jungleCulture"
          >
            <header>
              <PageTitle
                image="./images/team_design.jpg"
                imageAlignment="left"
                introText="Welkom tijdens jouw tweede week binnen Jungle Minds. Deze week staat in de teken van ons cultuur. Tijdens deze module leer je hoe wij ons cultuur gevormd hebben en hoe wij deze doorvertallen aan onze medewerkers."
                introTitle="Module 2"
                mediaCaption="Jungle Cultuur"
                pageTitle="2 De Jungle Cultuur"
                textMedia={
                  Array [
                    Object {
                      "text": "Tijdens deze week leer je meer over:
      <ul>
      <li>
        2.1 Jungle Formula
      </li>
      <li>
        2.2 Jungle Days
      </li>
      <li>
        2.3 ...
      </li>
      </ul>
      ",
                      "title": "Week 2",
                    },
                    Object {
                      "text": "Gedurende deze modules krijg je enkele opdrachten toegewegen. Het is de bedoeling dat je deze opdrachten uitvoert voor het einde van de week. Denk aan: de mandagen zijn de beste dagen om een gesprek aan te gaan met een collega. 
      <ul>
      <li>
        2.4 Opdrachten
      </li>
      </ul>
      ",
                      "title": "Opdrachten",
                    },
                  ]
                }
              />
              <Intro
                image="./images/team_design.jpg"
                imageAlignment="left"
                introText="Welkom tijdens jouw tweede week binnen Jungle Minds. Deze week staat in de teken van ons cultuur. Tijdens deze module leer je hoe wij ons cultuur gevormd hebben en hoe wij deze doorvertallen aan onze medewerkers."
                introTitle="Module 2"
                mediaCaption="Jungle Cultuur"
                pageTitle="2 De Jungle Cultuur"
                textMedia={
                  Array [
                    Object {
                      "text": "Tijdens deze week leer je meer over:
      <ul>
      <li>
        2.1 Jungle Formula
      </li>
      <li>
        2.2 Jungle Days
      </li>
      <li>
        2.3 ...
      </li>
      </ul>
      ",
                      "title": "Week 2",
                    },
                    Object {
                      "text": "Gedurende deze modules krijg je enkele opdrachten toegewegen. Het is de bedoeling dat je deze opdrachten uitvoert voor het einde van de week. Denk aan: de mandagen zijn de beste dagen om een gesprek aan te gaan met een collega. 
      <ul>
      <li>
        2.4 Opdrachten
      </li>
      </ul>
      ",
                      "title": "Opdrachten",
                    },
                  ]
                }
              />
            </header>
            <TextAndImage
              image="./images/team_design.jpg"
              imageAlignment="left"
              introText="Welkom tijdens jouw tweede week binnen Jungle Minds. Deze week staat in de teken van ons cultuur. Tijdens deze module leer je hoe wij ons cultuur gevormd hebben en hoe wij deze doorvertallen aan onze medewerkers."
              introTitle="Module 2"
              mediaCaption="Jungle Cultuur"
              pageTitle="2 De Jungle Cultuur"
              textMedia={
                Array [
                  Object {
                    "text": "Tijdens deze week leer je meer over:
      <ul>
      <li>
        2.1 Jungle Formula
      </li>
      <li>
        2.2 Jungle Days
      </li>
      <li>
        2.3 ...
      </li>
      </ul>
      ",
                    "title": "Week 2",
                  },
                  Object {
                    "text": "Gedurende deze modules krijg je enkele opdrachten toegewegen. Het is de bedoeling dat je deze opdrachten uitvoert voor het einde van de week. Denk aan: de mandagen zijn de beste dagen om een gesprek aan te gaan met een collega. 
      <ul>
      <li>
        2.4 Opdrachten
      </li>
      </ul>
      ",
                    "title": "Opdrachten",
                  },
                ]
              }
            />
          </Grid>,
        ],
        "className": "JungleCulture",
        "data-test": undefined,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "defer": true,
            "encodeSpecialCharacters": true,
            "title": "JungleCulture",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": Array [
              <header>
                <PageTitle
                  image="./images/team_design.jpg"
                  imageAlignment="left"
                  introText="Welkom tijdens jouw tweede week binnen Jungle Minds. Deze week staat in de teken van ons cultuur. Tijdens deze module leer je hoe wij ons cultuur gevormd hebben en hoe wij deze doorvertallen aan onze medewerkers."
                  introTitle="Module 2"
                  mediaCaption="Jungle Cultuur"
                  pageTitle="2 De Jungle Cultuur"
                  textMedia={
                    Array [
                      Object {
                        "text": "Tijdens deze week leer je meer over:
      <ul>
      <li>
        2.1 Jungle Formula
      </li>
      <li>
        2.2 Jungle Days
      </li>
      <li>
        2.3 ...
      </li>
      </ul>
      ",
                        "title": "Week 2",
                      },
                      Object {
                        "text": "Gedurende deze modules krijg je enkele opdrachten toegewegen. Het is de bedoeling dat je deze opdrachten uitvoert voor het einde van de week. Denk aan: de mandagen zijn de beste dagen om een gesprek aan te gaan met een collega. 
      <ul>
      <li>
        2.4 Opdrachten
      </li>
      </ul>
      ",
                        "title": "Opdrachten",
                      },
                    ]
                  }
                />
                <Intro
                  image="./images/team_design.jpg"
                  imageAlignment="left"
                  introText="Welkom tijdens jouw tweede week binnen Jungle Minds. Deze week staat in de teken van ons cultuur. Tijdens deze module leer je hoe wij ons cultuur gevormd hebben en hoe wij deze doorvertallen aan onze medewerkers."
                  introTitle="Module 2"
                  mediaCaption="Jungle Cultuur"
                  pageTitle="2 De Jungle Cultuur"
                  textMedia={
                    Array [
                      Object {
                        "text": "Tijdens deze week leer je meer over:
      <ul>
      <li>
        2.1 Jungle Formula
      </li>
      <li>
        2.2 Jungle Days
      </li>
      <li>
        2.3 ...
      </li>
      </ul>
      ",
                        "title": "Week 2",
                      },
                      Object {
                        "text": "Gedurende deze modules krijg je enkele opdrachten toegewegen. Het is de bedoeling dat je deze opdrachten uitvoert voor het einde van de week. Denk aan: de mandagen zijn de beste dagen om een gesprek aan te gaan met een collega. 
      <ul>
      <li>
        2.4 Opdrachten
      </li>
      </ul>
      ",
                        "title": "Opdrachten",
                      },
                    ]
                  }
                />
              </header>,
              <TextAndImage
                image="./images/team_design.jpg"
                imageAlignment="left"
                introText="Welkom tijdens jouw tweede week binnen Jungle Minds. Deze week staat in de teken van ons cultuur. Tijdens deze module leer je hoe wij ons cultuur gevormd hebben en hoe wij deze doorvertallen aan onze medewerkers."
                introTitle="Module 2"
                mediaCaption="Jungle Cultuur"
                pageTitle="2 De Jungle Cultuur"
                textMedia={
                  Array [
                    Object {
                      "text": "Tijdens deze week leer je meer over:
      <ul>
      <li>
        2.1 Jungle Formula
      </li>
      <li>
        2.2 Jungle Days
      </li>
      <li>
        2.3 ...
      </li>
      </ul>
      ",
                      "title": "Week 2",
                    },
                    Object {
                      "text": "Gedurende deze modules krijg je enkele opdrachten toegewegen. Het is de bedoeling dat je deze opdrachten uitvoert voor het einde van de week. Denk aan: de mandagen zijn de beste dagen om een gesprek aan te gaan met een collega. 
      <ul>
      <li>
        2.4 Opdrachten
      </li>
      </ul>
      ",
                      "title": "Opdrachten",
                    },
                  ]
                }
              />,
            ],
            "page": "jungleCulture",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <PageTitle
                    image="./images/team_design.jpg"
                    imageAlignment="left"
                    introText="Welkom tijdens jouw tweede week binnen Jungle Minds. Deze week staat in de teken van ons cultuur. Tijdens deze module leer je hoe wij ons cultuur gevormd hebben en hoe wij deze doorvertallen aan onze medewerkers."
                    introTitle="Module 2"
                    mediaCaption="Jungle Cultuur"
                    pageTitle="2 De Jungle Cultuur"
                    textMedia={
                      Array [
                        Object {
                          "text": "Tijdens deze week leer je meer over:
      <ul>
      <li>
        2.1 Jungle Formula
      </li>
      <li>
        2.2 Jungle Days
      </li>
      <li>
        2.3 ...
      </li>
      </ul>
      ",
                          "title": "Week 2",
                        },
                        Object {
                          "text": "Gedurende deze modules krijg je enkele opdrachten toegewegen. Het is de bedoeling dat je deze opdrachten uitvoert voor het einde van de week. Denk aan: de mandagen zijn de beste dagen om een gesprek aan te gaan met een collega. 
      <ul>
      <li>
        2.4 Opdrachten
      </li>
      </ul>
      ",
                          "title": "Opdrachten",
                        },
                      ]
                    }
                  />,
                  <Intro
                    image="./images/team_design.jpg"
                    imageAlignment="left"
                    introText="Welkom tijdens jouw tweede week binnen Jungle Minds. Deze week staat in de teken van ons cultuur. Tijdens deze module leer je hoe wij ons cultuur gevormd hebben en hoe wij deze doorvertallen aan onze medewerkers."
                    introTitle="Module 2"
                    mediaCaption="Jungle Cultuur"
                    pageTitle="2 De Jungle Cultuur"
                    textMedia={
                      Array [
                        Object {
                          "text": "Tijdens deze week leer je meer over:
      <ul>
      <li>
        2.1 Jungle Formula
      </li>
      <li>
        2.2 Jungle Days
      </li>
      <li>
        2.3 ...
      </li>
      </ul>
      ",
                          "title": "Week 2",
                        },
                        Object {
                          "text": "Gedurende deze modules krijg je enkele opdrachten toegewegen. Het is de bedoeling dat je deze opdrachten uitvoert voor het einde van de week. Denk aan: de mandagen zijn de beste dagen om een gesprek aan te gaan met een collega. 
      <ul>
      <li>
        2.4 Opdrachten
      </li>
      </ul>
      ",
                          "title": "Opdrachten",
                        },
                      ]
                    }
                  />,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "image": "./images/team_design.jpg",
                    "imageAlignment": "left",
                    "introText": "Welkom tijdens jouw tweede week binnen Jungle Minds. Deze week staat in de teken van ons cultuur. Tijdens deze module leer je hoe wij ons cultuur gevormd hebben en hoe wij deze doorvertallen aan onze medewerkers.",
                    "introTitle": "Module 2",
                    "mediaCaption": "Jungle Cultuur",
                    "pageTitle": "2 De Jungle Cultuur",
                    "textMedia": Array [
                      Object {
                        "text": "Tijdens deze week leer je meer over:
      <ul>
      <li>
        2.1 Jungle Formula
      </li>
      <li>
        2.2 Jungle Days
      </li>
      <li>
        2.3 ...
      </li>
      </ul>
      ",
                        "title": "Week 2",
                      },
                      Object {
                        "text": "Gedurende deze modules krijg je enkele opdrachten toegewegen. Het is de bedoeling dat je deze opdrachten uitvoert voor het einde van de week. Denk aan: de mandagen zijn de beste dagen om een gesprek aan te gaan met een collega. 
      <ul>
      <li>
        2.4 Opdrachten
      </li>
      </ul>
      ",
                        "title": "Opdrachten",
                      },
                    ],
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "image": "./images/team_design.jpg",
                    "imageAlignment": "left",
                    "introText": "Welkom tijdens jouw tweede week binnen Jungle Minds. Deze week staat in de teken van ons cultuur. Tijdens deze module leer je hoe wij ons cultuur gevormd hebben en hoe wij deze doorvertallen aan onze medewerkers.",
                    "introTitle": "Module 2",
                    "mediaCaption": "Jungle Cultuur",
                    "pageTitle": "2 De Jungle Cultuur",
                    "textMedia": Array [
                      Object {
                        "text": "Tijdens deze week leer je meer over:
      <ul>
      <li>
        2.1 Jungle Formula
      </li>
      <li>
        2.2 Jungle Days
      </li>
      <li>
        2.3 ...
      </li>
      </ul>
      ",
                        "title": "Week 2",
                      },
                      Object {
                        "text": "Gedurende deze modules krijg je enkele opdrachten toegewegen. Het is de bedoeling dat je deze opdrachten uitvoert voor het einde van de week. Denk aan: de mandagen zijn de beste dagen om een gesprek aan te gaan met een collega. 
      <ul>
      <li>
        2.4 Opdrachten
      </li>
      </ul>
      ",
                        "title": "Opdrachten",
                      },
                    ],
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
              ],
              "type": "header",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "image": "./images/team_design.jpg",
                "imageAlignment": "left",
                "introText": "Welkom tijdens jouw tweede week binnen Jungle Minds. Deze week staat in de teken van ons cultuur. Tijdens deze module leer je hoe wij ons cultuur gevormd hebben en hoe wij deze doorvertallen aan onze medewerkers.",
                "introTitle": "Module 2",
                "mediaCaption": "Jungle Cultuur",
                "pageTitle": "2 De Jungle Cultuur",
                "textMedia": Array [
                  Object {
                    "text": "Tijdens deze week leer je meer over:
      <ul>
      <li>
        2.1 Jungle Formula
      </li>
      <li>
        2.2 Jungle Days
      </li>
      <li>
        2.3 ...
      </li>
      </ul>
      ",
                    "title": "Week 2",
                  },
                  Object {
                    "text": "Gedurende deze modules krijg je enkele opdrachten toegewegen. Het is de bedoeling dat je deze opdrachten uitvoert voor het einde van de week. Denk aan: de mandagen zijn de beste dagen om een gesprek aan te gaan met een collega. 
      <ul>
      <li>
        2.4 Opdrachten
      </li>
      </ul>
      ",
                    "title": "Opdrachten",
                  },
                ],
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
